examples/in_cluster_config.py

```diff
@@ -12,47 +12,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Simple example to show loading config from the cluster
-#
-# It works only from a pod. You can start an image with Python
-# (for example python:latest), exec into the pod, install the library,
-# then try out this example.
-#
-# If you get 403 errors from API server you will have to configure
-# RBAC to add the permission to list pods.
-#
-# ---
-# kind: ClusterRole
-# apiVersion: rbac.authorization.k8s.io/v1
-# metadata:
-#   name: pods-list
-# rules:
-# - apiGroups: [""]
-#   resources: ["pods"]
-#   verbs: ["list"]
-# ---
-# kind: ClusterRoleBinding
-# apiVersion: rbac.authorization.k8s.io/v1
-# metadata:
-#   name: pods-list
-# subjects:
-# - kind: ServiceAccount
-#   name: default
-#   namespace: default
-# roleRef:
-#   kind: ClusterRole
-#   name: pods-list
-#   apiGroup: rbac.authorization.k8s.io
-# ---
-#
-# Doc: https://kubernetes.io/docs/reference/access-authn-authz/rbac/
+"""
+Showcases loading the Kubernetes config from within the cluster. This script
+must be run within a pod. You can start a pod with a Python image (for
```

Comment:
Is it within a pod or as a pod as said in the original comment?

Reply:
it runs in a container within a pod